### PR TITLE
Migration of Database using Liquibase.

### PR DESCRIPTION
**Pull request by Anish1631**

### CODE REVIEW

File: Liquibase_Setup.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  	`email` varchar(255) DEFAULT NULL,
+ 	`full_name` varchar(255) DEFAULT NULL,
+  	`mobile_number` varchar(255) DEFAULT NULL,
+  	`password` varchar(255) DEFAULT NULL,
+  	`role` tinyint DEFAULT NULL,
+  	`username` varchar(255) DEFAULT NULL,
+  	PRIMARY KEY (`id`),
+  	CONSTRAINT `users_chk_1` CHECK ((`role` between 0 and 1))
+	) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci
+```
```
Comment (sagarzond) [inline Liquibase_Setup.md]: No need to add this step

File: Liquibase_Setup.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+
+## Step 5: Make changes in the application.properties
+
+	spring.jpa.defer-datasource-initialization=false
+	spring.liquibase.change-log=classpath:Sql/changelog-master.xml
+	spring.liquibase.enabled=true
+
+Make sure the auto ddl update is off
+------------------------------------------------------------------------------------------------------------------------
```
Comment (sagarzond) [inline Liquibase_Setup.md]: No need to add this step

File: event_management/src/main/resources/Sql/changelog-master.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.2.xsd">
+    <include file="Sql/changelog-users-table.sql"/>
+    <include file="Sql/changelog-session-table.sql"/>
+    <include file="Sql/changelog-event_status-table.sql"/>
+    <include file="Sql/changelog-events-table.sql"/>
```
Comment (sagarzond) [inline event_management/src/main/resources/Sql/changelog-master.xml]: - updated package name from Sql to sql
- updated file name
- updated sequence of execution as event_status will need to have event table
```suggestion
    <include file="sql/create-users-table.sql"/>
    <include file="sql/create-session-table.sql"/>
    <include file="sql/create-events-table.sql"/>
    <include file="sql/create-event_status-table.sql"/>
```

File: Liquibase_Setup.md (hunk 1 above)
Comment (Anish1631) [reply Liquibase_Setup.md]: Done

File: Liquibase_Setup.md (hunk 2 above)
Comment (Anish1631) [reply Liquibase_Setup.md]: Done

File: event_management/src/main/resources/Sql/changelog-master.xml (hunk 3 above)
Comment (Anish1631) [reply event_management/src/main/resources/Sql/changelog-master.xml]: Done

**Review by sagarzond — APPROVED** [review]

**Pull request merged**